plugin/src/main/resources/org/wildfly/plugins/bootablejar/maven/cloud/openshift-clustering-script.cli

```diff
@@ -11,6 +11,14 @@ if (outcome == success) of /subsystem=jgroups/stack=tcp/protocol=FD_SOCK:read-re
   /subsystem=jgroups/stack=tcp/protocol=FD_SOCK:write-attribute(name=socket-binding
 end-if
 
+if (outcome == success) of /subsystem=jgroups/stack=udp/protocol=FD_SOCK2:read-resource
+ /subsystem=jgroups/stack=udp/protocol=FD_SOCK2:write-attribute(name=socket-binding
```

Comment:
Does the parenthesis need to be closed?  Maybe the command parser is not that strict?

Reply:
Yeah, that is not strictly required. I should fix it.